python/ray/dashboard/modules/metrics/dashboards/serve_llm_dashboard_panels.py

```diff
@@ -33,28 +33,28 @@
     Panel(
         id=2,
         title="vLLM: Time Per Output Token Latency",
-        description="Time per output token latency in seconds.",
-        unit="tokens",
+        description="Time per output token latency in milliseconds.",
+        unit="milliseconds",
         targets=[
             Target(
-                expr='histogram_quantile(0.99, sum by(le) (rate(ray_vllm:time_per_output_token_seconds_bucket{{model_name=~"$vllm_model_name", {global_filters}}}[5m])))',
-                legend="P99",
+                expr='1000 * histogram_quantile(0.99, sum by(le, model_name, WorkerId) (rate(ray_vllm:time_per_output_token_seconds_bucket{{model_name=~"$vllm_model_name", WorkerId=~"$workerid", {global_filters}}}[30s])))',
```

Comment:
nit: instead of doing manual conversion between ms and s, in grafana, you can set time units and I think it will automatically convert the output. 

Similar to how percent unit converts `0.35` into `35%`

Can you play around with that and see if that's better?